api/src/main/java/org/ehcache/config/CacheConfiguration.java

```diff
@@ -26,37 +26,43 @@
 /**
  * Represents the minimal read-only configuration for a Cache to be, or an already existing one
  *
+ * @param <K> the type of the keys used to access data within the cache
+ * @param <V> the type of the values held within the cache
+ *
  * @author Alex Snaps
  */
 public interface CacheConfiguration<K, V> {
 
   /**
-   * Not sure whether this should be exposed on this interface really
-   * @return unmodifiable collection of service configuration related to this cache
+   * Not sure whether this should be exposed on this interface really.
```

Comment:
Not necessarily part of this review but maybe we should take the time to answer this question? @alexsnaps  ?